lib/src/command_runner.dart

```diff
@@ -109,6 +112,13 @@ class PubCommandRunner extends CommandRunner<int> implements PubTopLevel {
     });
     argParser.addFlag('verbose',
         abbr: 'v', negatable: false, help: 'Shortcut for "--verbosity=all".');
+    argParser.addOption(
```

Comment:
Just curious, does it have to both be a top-level argument, and an argument in all the subcommands?

What happens if it's passed twice...

Reply:
Hmm - it doesn't have to. That just made more sense to me. I would personally instinctively give it at the top-level (it is an option that changes how the following command is interpreted), but it would be annoying to have not work for each command.

We could do it either way if you think it is better.